crates/ruff/src/resolver.rs

```diff
@@ -329,10 +329,14 @@ pub fn python_files_in_path(
                 }
             }
 
-            if result.as_ref().map_or(true, |entry| {
+            let should_include = result.as_ref().map_or(true, |entry| {
```

Comment:
Used a variable to avoid https://github.com/astral-sh/ruff/actions/runs/5561952462/jobs/10159951956

Reply:
I believe you can also avoid the clippy lint and retain the old pattern by not using `return` in your statement, see https://github.com/astral-sh/ruff/commit/c9b305e9cadbcc9af465643ba84ce13b460fc48d

Reply:
@zanieb Thanks! Updated